Utils/azuretools-core/src/com/microsoft/azuretools/ui/embeddedbrowser/JxBrowserUtil.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) Microsoft Corporation
+ *   <p/>
+ *  All rights reserved.
+ *   <p/>
+ *  MIT License
+ *   <p/>
+ *  Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+ *  documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+ *  the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+ *  to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+ *  <p/>
+ *  The above copyright notice and this permission notice shall be included in all copies or substantial portions of
+ *  the Software.
+ *   <p/>
+ *  THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO
+ *  THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ *  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+ *  TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ *  SOFTWARE.
+ */
+
+package com.microsoft.azuretools.ui.embeddedbrowser;
+
+import com.google.common.collect.ImmutableMap;
+
+public class JxBrowserUtil {
+    private static final ImmutableMap<String, String> versionMap = ImmutableMap.of(
+            "win32", "jxbrowser-win32-6.16",
+            "win64", "jxbrowser-6.16",
+            "mac", "jxbrowser-mac-6.16",
+            "linux64", "jxbrowser-linux64-6.16",
+            "linux32", "jxbrowser-linux64-6.16"
+            );
+
```

Comment:
May the JxBrowser factory method be added?

Reply:
Ok, I will add all of them and send out a new iteration

---
In reply to: [154283559](https://github.com/Microsoft/azure-tools-for-java/pull/1279#discussion_r154283559) [](ancestors = 154283559)

Reply:
Add simple method first. Will add more.

---
In reply to: [154285064](https://github.com/Microsoft/azure-tools-for-java/pull/1279#discussion_r154285064) [](ancestors = 154285064,154283559)